contentgrid-appserver-query-engine-impl-jooq/src/main/java/com/contentgrid/appserver/query/engine/jooq/JOOQThunkExpressionVisitor.java

```diff
@@ -201,24 +220,26 @@ private static void assertOneTerm(List<? extends ThunkExpression<?>> terms) thro
         }
     }
 
-    private static void assertTwoTerms(List<? extends ThunkExpression<?>> terms) throws InvalidThunkExpressionException {
+    private static void assertTwoTerms(List<? extends ThunkExpression<?>> terms)
+            throws InvalidThunkExpressionException {
         if (terms.size() != 2) {
             throw new InvalidThunkExpressionException("Operation requires 2 parameters.");
         }
     }
 
-    @Allow.PlainSQL
+    @PlainSQL
```

Comment:
I think it's clearer to have this called `@Allow.PlainSQL`.

Reply:
done